pkg/cmd/cli/describe/helpers.go

```diff
@@ -129,6 +129,7 @@ func FormatRelativeTime(t time.Time) string {
 
 func formatMeta(out *tabwriter.Writer, m api.ObjectMeta) {
 	formatString(out, "Name", m.Name)
+	formatString(out, "Namespace", m.Namespace)
```

Comment:
I don't want to print namespace unless we really have to - everything is effectively in a namespace.  Reason?